api/inheritance.md

```diff
@@ -0,0 +1,25 @@
+# Types
+
+[data](https://github.com/mdn/data/blob/master/api/inheritance.json) |
+[schema](https://github.com/mdn/data/blob/master/css/inheritance.schema.json)
+
+The inheritance files defines, for each interface, the parent that they inherit properties and methods from, as well as the mixins that they implements.
```

Comment:
Better:
`Interfaces of Web APIs can inherit from other interfaces or implement [mixins](https://developer.mozilla.org/en-US/docs/Glossary/Mixin). For each interface, this data informs about the inherited (parent) interface and the implemented mixin(s).`

Reply:
Fixed.